boards/arm/96b_carbon/doc/96b_carbon.rst

```diff
@@ -10,6 +10,32 @@ Zephyr applications use the 96b_carbon configuration to run on the 96Boards
 Carbon hardware. It is based on the STMicroelectronics STM32F401RET Cortex-M4
 CPU and also contains a nRF51832 chip connected over SPI for BLE connectivity.
 
+The 96Boards Carbon board is built with two chips: an STMicroelectronics
+STM32F401RET Cortex-M4 CPU and an nRF51832 chip connected to
```

Comment:
Thanks for the catch -- looks like that error is being propagated from the first introduction of the board. I'll fix all the instances, not just those introduced in this series.